### PR TITLE
Hiding the Regex group box

### PR DESCRIPTION
**Pull request by Ivanluv**

### CODE REVIEW

**Comment by Muthenya** [comment]
@Ivanluv what the current state of this? Is it ready for review or still in progress?

**Comment by Ivanluv** [comment]
@africanmathsinitiative/developers  this is ready for review

**Comment by Muthenya** [comment]
@Ivanluv kindly reference the pull request to the issue it is addressing.

File: instat/dlgStringHandling.vb [hunk 1]
```diff
@@ -90,6 +90,8 @@ Public Class dlgStringHandling
 
         'temporary disabling
         grpRegex.Enabled = False
```
Comment (Muthenya) [inline instat/dlgStringHandling.vb]: Just a thought. Since we are hiding the group box, could we remove the disabling of the group box?

**Comment by Muthenya** [comment]
This looks good from my end.

**Comment by Muthenya** [comment]
Could we 
1. Bring the receiver and textboxes up a bit?
2. Move the unseen radio buttons (regex and fixed) to come after the receiver and textboxes?

image

**Comment by dannyparsons** [comment]
Could the column receiver and pattern textbox also be the same width? They both look a bit longer than normal actually, but anyway it would look better if they have the same width.

**Comment by maxwellfundi** [comment]
@Ivanluv These looks good but look like it have very simple changes to make. Could you have a look at them.

**Comment by Ivanluv** [comment]
@maxwellfundi am on it

**Comment by maxwellfundi** [comment]
Tested this and its all good.

File: instat/dlgStringHandling.vb [hunk 2]
```diff
@@ -153,7 +155,7 @@ Public Class dlgStringHandling
         clsCountFunction.SetAssignTo(ucrSaveStringHandling.GetText, strTempDataframe:=ucrSelectorStringHandling.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:=ucrSaveStringHandling.GetText, bAssignToIsPrefix:=True)
         ucrBase.clsRsyntax.SetBaseRFunction(clsCountFunction)
         NewColumnName()
-        ChangeSize()
+        'ChangeSize()
```
Comment (dannyparsons) [inline instat/dlgStringHandling.vb]: Why is this commented out? And also below.

Comment (dannyparsons) [reply instat/dlgStringHandling.vb]: Ok that makes sense. So if the method is no longer needed it should be deleted along with all calls to it.

**Pull request merged**